src/posts/2021-08-14-reviewing-a-design-for-accesibility.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Reviewing a design for accessibility
+description: "Reviewing designs for accessibility is a great way to help designers think more accessibly and to help developers implement more accessibly. We can use short checklists to help us make sure we haven't missed any big things."
+category: Quick tip
+author: Steve Barnett
+date: 2021-08-14
+tags:
+  - quicktip
+---
+
+I'm a big believer in [shifting left](https://www.deque.com/shift-left/): considering accessibility as early in the process as we can. One way we can do this is by reviewing designs for accessibility. This helps us catch anything we've missed before we start developing it.
+
+## What to do: annotate
+
+By adding annotations about the semantics and accessibility of elements on a page we can remove barriers to access. The best way to do annotations might be sticky notes on a printout or comments on a Figma file, or something else. The important thing is that we want to be clear and explicit.
+
+Here are a handful of things that I've found give good value / effort balance when reviewing a design for accessibility. 
+
+## Things to check
+
+- **Page title**
+    - Is the page title unique?
+    - Does it describe the page content?
+- **Headings**
+    - Are headings used liberally?
+    - Are the levels shown, without skipping any?
+- **Text alternatives**
+    - Do all images (including icons) have a text alternative? 
+    - If the image is decorative, is it marked as such? Are we sure it's only decorative?
+- **Link and button text**
+    - Does each link describe where we'll go when we follow it? Does each button describe what will happen when we use it?
+    - Where links or buttons have the same text repeated on a page, do they have a longer accessible name that is unique? For example: "View details of Quarter 1 review" instead of "View details".
+- **Standard controls**
+    - Are standard controls used? They have accessibility baked in. Example: a pair of radio buttons or a checkbox instead of a custom toggle.
+- **Names for form elements**
+    - Do all form controls (one input plus one label) have a name? 
```

Comment:
Could we link to a post about how to create the input/label association?

Reply:
Yes! Added in 98c5964 :)